core/src/trezor/workflow.py

```diff
@@ -97,22 +97,6 @@ def set_default(constructor: Callable[[], loop.Task]) -> None:
     default_constructor = constructor
 
 
-def kill_default() -> None:
```

Comment:
please keep and hide behind `if False`

Reply:
Done in [ccdccd9](https://github.com/trezor/trezor-firmware/pull/2525/commits/ccdccd9eee7c79ac232a344004703483d2717db6)